modules/ztp-configuring-disk-partitioning.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_module-type: PROCEDURE
+[id="ztp-configuring-disk-partitioning_{context}"]
+= Configuring disk partitioning with SiteConfig
+
+You can use the `SiteConfig` YAML file to generate the `MachineConfig` CR used for disk partitioning. You need to to modify values in the `MachineConfig` CR to reflect dependencies on the underlying disk.
```

Comment:
`You need to to modify values in the MachineConfig CR to reflect dependencies on the underlying disk.`  - imo this statement is not clear. The user entrypoint is the SiteConfig and that's where the disk related values are added.  The user doesn't edit MachineConfig directly as they get generated from the SiteConfig.

Reply:
Done.